apps/smart-contracts/core/test/DepositHook.test.ts

```diff
@@ -342,4 +369,48 @@ describe('=> DepositHook', () => {
       await expect(tx).to.emit(depositHook, 'AllowlistChange').withArgs(mockAllowlist.address)
     })
   })
+
+  describe('# setRequiredScore', () => {
+    it('reverts if not role holder', async () => {
+      expect(
+        await depositHook.hasRole(await depositHook.SET_REQUIRED_SCORE_ROLE(), user.address)
+      ).to.eq(false)
+
+      await expect(depositHook.connect(user).setRequiredScore(0)).revertedWith(
+        `AccessControl: account ${user.address.toLowerCase()} is missing role ${await depositHook.SET_REQUIRED_SCORE_ROLE()}`
+      )
+    })
+
+    it('sets required score > 0', async () => {
+      expect(await depositHook.getRequiredScore()).to.eq(0)
```

Comment:
```suggestion
      expect(await depositHook.getRequiredScore()).to.not.be.gt(0)
```
Should reflect name of testcase.

Reply:
same for the expect statement (`to.be.gt(0)`), and same for the other tests below